src/Magnum/Math/Geometry/Intersection.h

```diff
@@ -123,8 +125,74 @@ class Intersection {
             const T f = dot(planePosition, planeNormal);
             return (f-dot(planeNormal, p))/dot(planeNormal, r);
         }
+
+        /**
+         * @brief Intersection of a point and a camera frustum
+         * @param point Point
+         * @param frustum Frustum planes with normals pointing outwards
+         * @return `true` if the point is on or inside the frustum.
+         *
+         * @todoc formulas used
+         */
+        template<class T> bool pointFrustum(const Vector3<T>& point, const Vector4<T> (&frustum)[6]) {
+            for(const Vector4<T>& f : frustum) {
+                if(pointPlaneHalfSpace(point, f) < 0) {
+                    /* the point is in front of one of the frustum planes (normals point outwards) */
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
+        /**
+         * @brief Intersection of a range and a camera frustum
+         * @param box Range
+         * @param frustum Frustum planes with normals pointing outwards
+         * @return `true` if the box intersects with the camera frustum.
+         */
+        template<class T> bool rangeFrustum(const Range3D<T>& box, Containers::StaticArrayView<6, const Vector4<T>> frustum);
```

Comment:
I agree with having the `Frustum` class. And I also agree with 

> So I would keep them [the intersection functions] here [In Intersections.h] I think.

since it is one consistent place to look for intersection functions. If `pointLine, pointPlane, pointAABB etc` were all in that namespace, but `pointFrustum` not, that would cause a world of confusion.

Regarding batch processing: I will skip that for now (you could put it in Projects :) ), since that will most likely be an additional method.